custom/conf/app.example.ini

```diff
@@ -2602,6 +2622,9 @@ ROUTER = console
 ; [actions]
 ;; Enable/Disable actions capabilities
 ;ENABLED = false
+;;
+;STORAGE_TYPE = local
+;;
```

Comment:
Other sections have newly added `MINIO_BASE_PATH` key, this `actions` section doesn't?